client/lib/mixins/i18n/index.js

```diff
@@ -14,7 +14,7 @@ var debug = require( 'debug' )( 'calypso:i18n' ),
 var config = require( 'config' ),
 	numberFormatPHPJS = require( './number-format' ),
 	emitter = require( 'lib/mixins/emitter' ),
-	interpolateComponents = require( 'lib/interpolate-components' );
+	interpolateComponents = require( 'lib/interpolate-components/src/' );
```

Comment:
> By providing th full path the component will be compiled as part of the parent projects build process.

I'm not sure I follow that. By requiring directly into the component's inner-files, we're just requiring the pre-compiled `/src/index.js` file. Couldn't we just build the module as part of calypso's build process and then require it by name and then allow it to bring in the specified main file?

Or if we just want to require the original source file, could we change the location of `interpolate-components` inside the main package.json file to:
```js
"interpolate-components": "client/lib/interpolate-components/src",
```